Arduino_Sub_DAQ/Arduino_Sub_DAQ.ino

```diff
@@ -0,0 +1,89 @@
+/*
+* Linear Potentiometers
+* Author:        Bazel Jazzar   Suspension Lead
+* Modified By:   Manuel De Jesus Contreras & Cristian Cruz    Embedded Firmware Team
+* Email:         Manuel.DeJesusContreras@UTDallas.edu
+* 
+* (c) 2022 Dallas Formula Racing - Formula SAE
+* GPL-3.0 License
+*/
+
+
+#include <SPI.h>
+#include <SD.h>
+File my_file;
+
+bool hasnt_close = true;
```

Comment:
Thoughts on naming this `has_not_closed` instead?

Reply:
I think that works better, will fix.